tests/unit/helpers/format-message-test.js

```diff
@@ -179,7 +179,7 @@ test('able to discover all register translations', function(assert) {
   assert.expect(1);
   this.intl.addTranslation('es_MX', 'foo', 'bar'); /* tests that the locale name becomes normalized to es-mx */
   this.intl.exists('test', 'fr-ca');
-  assert.equal(this.intl.get('locales').join('; '), 'en-us; es-es; fr-fr; es-mx');
```

Comment:
I don't have a solution for this yet, but it may involve determine locales that have been bundled at build time and making that list available as `ENV.intl.bundledLocales`.